src/components/Event/docs/EventWiringExample.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Event } from '../';
+import styles from './EventWiringExample.st.css';
+
+export const EventWiringExample = () => {
+  return (
+    <div {...styles('flexContainer')}>
+      <div>
+        <h4>Regular</h4>
+        <Event {...styles('root')} time="20:30" title="Blink 182 Concert" />
+      </div>
+      <div>
+        <h4>All Day</h4>
+        <Event
+          {...styles('root')}
+          isMultiday
+          time="20:30"
+          title="Blink 182 Concert"
+        />
+      </div>
+    </div>
```

Comment:
Even though it seems to be working, the correct stylable syntax would be:
```suggestion
    <div {...styles('root')}>
      <div>
        <h4>Regular</h4>
        <Event className={styles.event} time="20:30" title="Blink 182 Concert" />
      </div>
      <div>
        <h4>All Day</h4>
        <Event
          className={styles.event}
          isMultiday
          time="20:30"
          title="Blink 182 Concert"
        />
      </div>
    </div>
```

Reply:
event doesn't have className in it's props. it should have ? anyway, i updated the names